### ISSUE TITLE
java.lang.IllegalArgumentException: Key "TCS3400 RGB_CT" was already used.

### ISSUE THREAD

**Issue by chaoscalm** [kamgurgul/cpu-info#103]
[crash-com-kgurgul-cpuinfo-14_12-00-23-32_441.zip](https://github.com/kamgurgul/cpu-info/files/13666673/crash-com-kgurgul-cpuinfo-14_12-00-23-32_441.zip)

```
FATAL EXCEPTION: main
Process: com.kgurgul.cpuinfo, PID: 4549
java.lang.IllegalArgumentException: Key "TCS3400 RGB_CT" was already used. If you are using LazyColumn/Row please make sure you provide a unique key for each item.
at l1.a0.A(SourceFile:177)
at l1.a0$c.l0(SourceFile:8)
at u.x.Y0(SourceFile:36)
at t.v.b(SourceFile:17)
at t.s.e(SourceFile:18)
at t.q$c.a(SourceFile:443)
at t.q$c.d0(SourceFile:9)
at u.v$a$a.a(SourceFile:19)
at u.v$a$a.d0(SourceFile:9)
at l1.a0$d.a(SourceFile:140)
at n1.w.g(SourceFile:56)
at o.b$b.a(SourceFile:11)
at o.b$b.a0(SourceFile:11)
at l1.z.d(SourceFile:17)
at n1.e0.g(SourceFile:12)
at o.b$a.a(SourceFile:11)
at o.b$a.a0(SourceFile:11)
at l1.z.d(SourceFile:17)
at n1.e0.g(SourceFile:12)
at androidx.compose.ui.graphics.f.d(SourceFile:11)
at n1.e0.g(SourceFile:12)
at androidx.compose.foundation.layout.f.d(SourceFile:111)
at n1.e0.g(SourceFile:12)
at n1.n0$d.a(SourceFile:9)
at n1.n0$d.B(SourceFile:1)
at r0.h$a.d(SourceFile:70)
at r0.w$a.g(SourceFile:61)
at r0.w.n(SourceFile:35)
at n1.j1.h(SourceFile:18)
at n1.j1.f(SourceFile:27)
at n1.n0.Q(SourceFile:31)
at n1.n0.h(SourceFile:1)
at n1.n0$b.s1(SourceFile:129)
at n1.i0.X0(SourceFile:20)
at n1.s0.g(SourceFile:4)
at n1.s0.p(SourceFile:63)
at androidx.compose.ui.platform.AndroidComposeView.f(SourceFile:13)
at n1.i0.e(SourceFile:31)
at t.a0.x(SourceFile:54)
at t.a0$g.a(SourceFile:4)
at t.a0$g.j0(SourceFile:7)
at p.g$b.a(SourceFile:19)
at androidx.compose.foundation.gestures.e$a.a(SourceFile:36)
at androidx.compose.foundation.gestures.e$a.j0(SourceFile:7)
at o.a.c(SourceFile:233)
at androidx.compose.foundation.gestures.e.a(SourceFile:27)
at androidx.compose.foundation.gestures.e$c$a.a(SourceFile:15)
at androidx.compose.foundation.gestures.e$c$a.j0(SourceFile:7)
at androidx.compose.foundation.gestures.e$c$b.a(SourceFile:13)
at p.f$a$a.a(SourceFile:23)
at p.f$a$a.j0(SourceFile:3)
at n.c1.k(SourceFile:38)
at n.c1.l(SourceFile:28)
at n.c1.a(SourceFile:1)
at n.c1$f.a(SourceFile:20)
at n.c1$f.j0(SourceFile:7)
at n.c1$h.a(SourceFile:7)
at n.c1$h.j0(SourceFile:7)
at i0.h$a.b(SourceFile:11)
at i0.h.l(SourceFile:25)
at i0.g2$k$a.a(SourceFile:27)
at i0.g2$k$a.j0(SourceFile:7)
at androidx.compose.ui.platform.j0$c.doFrame(SourceFile:11)
at androidx.compose.ui.platform.h0.w0(SourceFile:34)
at androidx.compose.ui.platform.h0.q0(SourceFile:1)
at androidx.compose.ui.platform.h0$d.doFrame(SourceFile:17)
at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1229)
at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1239)
at android.view.Choreographer.doCallbacks(Choreographer.java:899)
at android.view.Choreographer.doFrame(Choreographer.java:827)
at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1214)
at android.os.Handler.handleCallback(Handler.java:942)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loopOnce(Looper.java:201)
at android.os.Looper.loop(Looper.java:288)
at android.app.ActivityThread.main(ActivityThread.java:7924)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
Suppressed: x6.h: [androidx.compose.ui.platform.l1@79d6198, i0.h@54f0df1, g2{Cancelling}@24e8dd6, h0@42dc857]

**Comment by kamgurgul** [comment]
Is fixed in 5.0.0

**Comment by chaoscalm** [comment]
Very good, tested and no more crash. Thank you @kamgurgul
